Core/RestaurantReservation.Application/Services/AuthenticationService.cs

```diff
@@ -0,0 +1,21 @@
+﻿using RestaurantReservation.Application.Contracts.IServices;
```

Comment:
Still need to review The services here, then the domain, the the Persistance and Infrastructure.